packages/explorer/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
this isn't documented anywhere but I tend to not use import path aliases as it makes refactoring with IDE really bad, where moving a file causes all sorts of weird and unpredictable behavior

I tend to prefer relative imports instead

Reply:
I personally prefer aliases but not fixated on this, and good point about refactoring. Changed to relative paths